test/integration/targets/module_utils_Ansible.Basic/library/ansible_basic_tests.ps1

```diff
@@ -155,6 +155,7 @@ $tests = @{
         "_ansible_shell_executable": "ignored",
         "_ansible_socket": "ignored",
         "_ansible_syslog_facility": "ignored",
+		"_ansible_target_log_info": "ignored",
```

Comment:
indentation issue?

Reply:
this one is weird, didn't show as such, but did not pass sanity,  but jordan fixed  .. now it shows 'wrong' but passes tests